### PR TITLE
build(deps): fix sample audit issues

### PR DESCRIPTION
**Pull request by BenLiuMattr**
## Description

[DXM-520](https://mattrglobal.atlassian.net/browse/DXM-520)

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] The commit message(s) follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Documentation has been added / updated (for bug fixes / features)
- [x] Changes follow the **[contributing](../docs/CONTRIBUTING.md)** document.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Which merge strategy will you use?

- [x] Squash
- [ ] Rebase (REVIEW COMMITS)

### CODE REVIEW

File: sample/browser/index.web-sample.js [hunk 1]
```diff
@@ -28,25 +28,27 @@ import bbsContext from "./data/bbs.json";
...
 
 /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
 const documents = {
   "did:example:489398593#test": keyPairOptions,
   "did:example:489398593": exampleControllerDoc,
   "https://w3id.org/security/bbs/v1": bbsContext,
   "https://w3id.org/citizenship/v1": citizenVocab,
-  "https://www.w3.org/2018/credentials/v1": credentialContext
+  "https://www.w3.org/2018/credentials/v1": credentialContext,
+  "https://w3id.org/security/suites/jws-2020/v1": suiteContext,
```
Comment (BenLiuMattr) [inline sample/browser/index.web-sample.js:40]: To cache this document context to fix the following error when running `demo`

```
/Users/ben/code/jsonld-signatures-bbs/node_modules/jsonld/lib/ContextResolver.js:173
      throw new JsonLdError(
ContextResolver.js:173 Uncaught (in promise) jsonld.InvalidUrl: Dereferencing a URL did not result in a valid JSON-LD object. Possible causes are an inaccessible URL perhaps due to a same-origin policy (ensure the server uses CORS if you are using client-side JavaScript), too many redirects, a non-JSON response, or more than one HTTP Link Header was provided for a remote context.
    at ContextResolver._fetchContext (webpack:////Users/ben/code/jsonld-signatures-bbs/node_modules/jsonld/lib/ContextResolver.js?:173:13)
    at async ContextResolver._resolveRemoteContext (webpack:////Users/ben/code/jsonld-signatures-bbs/node_modules/jsonld/lib/ContextResolver.js?:117:34)
    at async ContextResolver.resolve (webpack:////Users/ben/code/jsonld-signatures-bbs/node_modules/jsonld/lib/ContextResolver.js?:50:22)
    at async api.process (webpack:////Users/ben/code/jsonld-signatures-bbs/node_modules/jsonld/lib/context.js?:66:20)
    at async jsonld.frame (webpack:////Users/ben/code/jsonld-signatures-bbs/node_modules/jsonld/lib/jsonld.js?:462:21)
    at async BbsBlsSignatureProof2020.getVerificationMethod (webpack:////Users/ben/code/jsonld-signatures-bbs/lib/BbsBlsSignatureProof2020.js?:313:24)
    at async BbsBlsSignatureProof2020.deriveProof (webpack:////Users/ben/code/jsonld-signatures-bbs/lib/BbsBlsSignatureProof2020.js?:140:36)
    at async deriveProof (webpack:////Users/ben/code/jsonld-signatures-bbs/lib/deriveProof.js?:50:20)
    at async main (webpack:///./index.web-sample.js?:109:24)
```

**Review by barendb — APPROVED** [review]
LGTM

File: sample/browser/package.json [hunk 2]
```diff
@@ -10,14 +10,21 @@
...
   },
   "dependencies": {
-    "@mattrglobal/jsonld-signatures-bbs": "0.10.0",
+    "@mattrglobal/jsonld-signatures-bbs": "link:../../",
     "jsonld-signatures": "5.0.1"
+  },
+  "resolutions": {
+    "webpack/**/glob-parent": "5.1.2",
+    "webpack-dev-server/sockjs-client/eventsource": "1.1.1",
+    "@mattrglobal/jsonld-signatures-bbs/**/node-forge": "1.3.0",
```
Comment (ZedMattr) [inline sample/browser/package.json:26]: This should be fixed in the main repo?

Comment (BenLiuMattr) [reply sample/browser/package.json:26]: Had a chat with Zed, keep the sample resolutions atm, likely the `resolutions` in main package not work in sample if we use link to the package.

Will update to use specific version after the main package published.

**Review by ZedMattr — APPROVED** [review]

**Pull request merged**